### PR TITLE
[REEF-533] Add missing Javadoc comments in Java code: reef-utils

### PR DESCRIPTION
**Pull request by tcNickolas**
JIRA:
  [REEF-553](https://issues.apache.org/jira/browse/REEF-533)

Pull Request:
  Closes #

### CODE REVIEW

**Comment by tcNickolas** [comment]
Second commit fixes link in commit message (JIRA number)

File: lang/java/reef-utils/src/main/java/org/apache/reef/util/package-info.java [hunk 1]
```diff
@@ -17,6 +17,6 @@
  * under the License.
  */
 /**
- * Utilities used across reef modules. Subpackages are structred following the java.utils package.
+ * Utilities used across reef modules. Subpackages are structured following the java.utils package.
```
Comment (dafrista) [inline lang/java/reef-utils/src/main/java/org/apache/reef/util/package-info.java]: The second sentence doesn't seem true. Can we remove it?

**Comment by dafrista** [comment]
I made a pass with one comment.

**Comment by tcNickolas** [comment]
I've updated PR

**Comment by dafrista** [comment]
I'll test and merge.

**Pull request closed without merging**